TMDBApp/TMDBApp/Genrer.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Genrer.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 15/01/2017.
+//  Copyright © 2017 Richard Leh. All rights reserved.
+//
+
+import Foundation
+
+enum ServerTMDBGenrer:String{
+    case id = "id"
+    case name = "name"
+}
+
+class Genrer{
```

Comment:
When you are beginning a project in Swift, do you prefer to adopt struct or class when you don't know yet how the project is going to grow?